cli/src/forge.rs

```diff
@@ -27,7 +28,39 @@ mod utils;
 #[tracing::instrument(err)]
 fn main() -> eyre::Result<()> {
     utils::subscriber();
-
+    if cfg!(target_os = "windows") {
+        println!("Can't source env files on windows");
+    } else {
+        let home: String = env::var("HOME")?;
+        let paths = [
+            PathBuf::from(&home),
+            PathBuf::from(&home).join(".foundry"),
+            PathBuf::from(&home).join(".config/foundry"),
+        ];
+        let files = [String::from(".dapprc"), String::from(".forgerc")];
+        let mut flag = false;
+        for path in paths.iter() {
+            for file in &files {
+                let arg = PathBuf::from(&path).join(file);
+                match std::process::Command::new("source").args(arg.to_str()).spawn() {
```

Comment:
isn't source a bash command? so we would need to run `bash -c source {}` instead?

Reply:
Yeah, sorry about not putting the whole context. So, apparently, the correct command is `. .dapprc` in order to export the env variables when called from an external program. 

The issue that I have (for unknown reasons still) is that while `cat` will work as expected, finding the source file and showing it's content, `source` and `.` will output a `file not found` error. 

I will have to dig in this. A solution we thought we @gakonst is perhaps using a crate where we can define a bash script as a literal string and execute that script, which will do the exports for us. 

Let me do the investigation tonight or tomorrow morning and come back with results.